src/main/msp/msp_protocol.h

```diff
@@ -60,7 +60,7 @@
 #define MSP_PROTOCOL_VERSION                0
 
 #define API_VERSION_MAJOR                   1 // increment when major changes are made
-#define API_VERSION_MINOR                   26 // increment when any change is made, reset to zero when major changes are released after changing API_VERSION_MAJOR
+#define API_VERSION_MINOR                   27 // increment when any change is made, reset to zero when major changes are released after changing API_VERSION_MAJOR
```

Comment:
@azolyoung you can check for version 1.7.3 and above. Current stable release is 1.7.2 so this won't affect anything but development branch.